src/test/regress/sql/with.sql

```diff
@@ -1065,3 +1065,36 @@ WITH cte1 AS (
 SELECT * FROM cte1 a JOIN cte1 b USING (c1);
 
 DROP TABLE t1;
+
+-- Test join prefetch_inner in the case of bottleneck join
+SET optimizer = off;
+--start_ignore
+DROP TABLE IF EXISTS d;
+--end_ignore
+CREATE TABLE d (c1 int, c2 int) DISTRIBUTED BY (c1);
+
+INSERT INTO d (VALUES ( 2, 0 ),( 2 , 0 ));
+
+-- Test with HashJoin
+WITH cte AS (
```

Comment:
Could you use `explain (analyze, costs off)` for three tests below to see plan?

Reply:
will the time values affect the passing of the test?

Reply:
no, many test are using explain analyze and differing time is filtered

Reply:
for the correct processing of the explain analyze results, it is necessary to include this in the test. is it worth it or just do explain + execution or just execution? https://github.com/arenadata/gpdb/blob/3756bbba81d9e01ee5a8b4d2238734c47fa0c43e/src/test/regress/sql/explain_analyze.sql#L1-L22